src/config/gmail.php

```diff
@@ -41,6 +41,23 @@
 		'metadata',
 	],
 
+    /*
+	|--------------------------------------------------------------------------
+	| Additional Scopes [URL Style]
+	|--------------------------------------------------------------------------
+	|
+	|   'additional_scopes' => [
+	|        'https://www.googleapis.com/auth/drive',
+	|        'https://www.googleapis.com/auth/documents'
+	|   ],
+	|
+	|
+	*/
+
+    'additional_scopes' => [
```

Comment:
I don't understand this part here, why did you add `additional_scopes` when you have `scopes` already.

Reply:
https://github.com/dacastro4/laravel-gmail/pull/65#issuecomment-486810224